zitadel/terraform/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+jwt_profile_file = "local-token.json"
+redirect_xpanse-ui_uris    = ["http://localhost:3000/authentication/callback"]
+post_logout_redirect_uris   = ["http://localhost:3000","http://localhost:3000/credentials","http://localhost:3000/home","http://localhost:3000/register","http://localhost:3000/monitor","http://localhost:3000/","http://localhost:3000/catalog","http://localhost:3000/services","http://localhost:3000/myServices","http://localhost:3000/order","http://localhost:3000/createService"]
+redirect_swagger-ui_uris   = ["http://localhost:8080/swagger-ui/oauth2-redirect.html"]
```

Comment:
same here.. we need only /swagger-ui/oauth2-redirect.html here. 
the base_swagger_url must be in environment related var file.

Reply:
fixed